plugins/s3_auth/unit_tests/test_aws_auth_v4.cc

```diff
@@ -392,7 +397,7 @@ TEST_CASE("AWSAuthSpecByExample: GET Object", "[AWS][auth][SpecByExample]")
     "host;range;x-amz-content-sha256;x-amz-date",
   };
 
-  ValidateBench(api, /*signePayload */ true, &now, bench, defaultIncludeHeaders, defaultExcludeHeaders);
+  ValidateBench(api, /*signPayload */ true, &now, bench, defaultIncludeHeaders, defaultExcludeHeaders);
```

Comment:
I am willing to split these code comment changes into a separate PR if that's what we want.